src/OrchardCore/OrchardCore.Data.YesSql.Abstractions/YesSqlOptions.cs

```diff
@@ -19,4 +19,6 @@ public class YesSqlOptions
     public IContentSerializer ContentSerializer { get; set; }
 
     public string TablePrefixSeparator { get; set; } = "_";
+
+    public IdentityColumnSize IdentityColumnType { get; set; } = IdentityColumnSize.Int32;
```

Comment:
@hishamco 

I think we should remove this from this PR. This can cause a long debate and should not be blocking to this PR.

When this option is added, the value would have to be stored in the `ShellSettings` for each tenant specially when the value is changed to `Int64`. otherwise tenants with long values in their databases could have overflow exception if the options changed again.

IdentitySize + schema option would be part of a separate PR after releasing 1.5.

Reply:
Furthermore I think the name `IdentityColumnSize` is not suited